static/src/stylesheets/module/content/_badging.scss

```diff
@@ -61,3 +88,39 @@
     }
 }
 
+/* AUS Election Liveblog Image
+   ========================================================================== */
+.content__head--aus-election {
```

Comment:
Yeah most of this is duplicating what we have above. Not sure we need to create a variant especially as this looks like it's the same but with a different image